reana/reana_dev/release.py

```diff
@@ -84,10 +85,21 @@ def release_commands():
 )
 @click.option("--user", "-u", default="reanahub", help="DockerHub user name [reanahub]")
 @click.option("--image-name", help="Should the component have a custom image name?")
+@click.option(
+    "--registry", "-r", default="docker.io", help="Registry to use in the image tag"
```

Comment:
It would be good to specify in the help string `[default=docker.io]` explicitly, as we do for `[reanahub]` the user name.